pandas/io/tests/parser/common.py

```diff
@@ -1635,3 +1635,12 @@ def test_file_handles(self):
                 if PY3:
                     self.assertFalse(m.closed)
                 m.close()
+
+    def test_file_system_encoding(self):
+        """
```

Comment:
don't use a comment here with triple-quotes, instead use ``#``
add the issue number as a comment